### DISCUSSION TITLE
现成的包

### DISCUSSION THREAD

**Discussion by zhujiwen** [apache/doris#6693] [RFC]
我通过docker打成的包一直无法正常运行，谁可以提供一个现成的编译好的，在此感谢

**Comment by pushiwuhua7** [comment]
可以下百度的palo，都是一个东西

**Reply by zhujiwen** [reply]
好的 多谢

**Comment by hf200012** [comment]
http://palo.baidu.com/docs/%E4%B8%8B%E8%BD%BD%E4%B8%93%E5%8C%BA/%E9%A2%84%E7%BC%96%E8%AF%91%E7%89%88%E6%9C%AC%E4%B8%8B%E8%BD%BD/
You can download the pre-compiled installation package here

**Reply by zhujiwen** [reply]
This is very helpful to me. Thank you very much
